server/main.go

```diff
@@ -56,6 +57,21 @@ func main() {
 		}
 	}
 
+	ddAgentAddr := os.Getenv("DD_AGENT_ADDR")
+	if ddAgentAddr == "" {
+		ddAgentAddr = "dd-agent.default.svc.cluster.local:8125"
+	}
+
+	env := os.Getenv("ENVIRONMENT")
+
+	client, err := statsd.New(ddAgentAddr)
+	if err != nil {
+		fatal(logger, err)
+	}
+	tags := []string{fmt.Sprintf("env:%s", env), "service:admission-control", "namespace:admission-control"}
```

Comment:
i know you're adding a service tag but since 1000 other metrics we're pushing scope the metric by name instead of tag it's probably a good idea to be consistent with the existing pattern